redfish/resource_redfish_bios.go

```diff
@@ -102,6 +101,16 @@ func getResourceRedfishBiosSchema() map[string]*schema.Schema {
 			}, false),
 			Default: string(redfish.GracefulRestartResetType),
 		},
+		"reset_timeout": {
```

Comment:
Do we want to set the default values of reset_timeout and bios_job_timeout to defaultBiosConfigServerResetTimeout and defaultBiosConfigJobTimeout respectively?